html/pfappserver/root/static.alt/src/components/pfMixinEscapeKey.js

```diff
@@ -0,0 +1,20 @@
+export default {
+  name: 'MixinEscapeKey',
+  methods: {
+    close () {
+      this.$router.push({ name: 'roles' })
```

Comment:
As the default behaviour, it should throw an exception when in *debug mode* (`process.env.VUE_APP_DEBUG`).